doc/user_guide/dynamo0p3.rst

```diff
@@ -256,8 +256,9 @@ basis/differential-basis functions required by the kernel are to be evaluated.
 Stencils
 ++++++++
 
-Kernel metadata may specify that a Kernel performs a stencil operation
-on a field. Any such metadata must provide a stencil type. See the
+The metadata for a Kernel which operates on a cell-column may specify
```

Comment:
Yes, I was torn between the two options. I've now added separate sections (for the metadata rules and the argument-list rules).

Reply:
I still think there is too little explanation at the moment. The user will want to know when to use domain and cell_column and why they are used and why there are restrictions - if any. I also think it might be better to restructure at some point. It does not seem right to introduce all the metadata options and then say domain supports a subset. However, seeing as we're just adding metadata support, this can be left until the actual implementation.